src/main/java/com/agency04/sbss/pizza/PizzaDeliveryService.java

```diff
@@ -0,0 +1,56 @@
+package com.agency04.sbss.pizza;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+@Qualifier("deliveryService")
+public class PizzaDeliveryService  implements PizzeriaService  {
```

Comment:
Za delivery service bi trebao postojat interface PizzeriaDeliveryService koji ima exposanu metodu orderPizza. Onda ce ga ova klasa implementirat, npr. `public class PizzaDeliveryServiceImpl implements PizzaDeliveryService`. Sve sta ce bit potrebno u toj klasi je injectat jedan od pizzeria service-a na neki od 3 nacina po izboru ka sta pise u zadatku: `injection method of you choosing (constructor, setter or field) to inject the dependency into PizzaDeliveryService`i definirat orderPizza metodu npr ovako:
```
 @Override
    public String orderPizza(Pizza pizza) {
        return pizzeriaService.makePizza(pizza) + "\nYour order should be arriving soon...";
    }
```